python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -489,7 +489,9 @@ def pending_processor_usage(self) -> ExecutionResources:
         raise NotImplementedError
 
     @abstractmethod
-    def base_resource_usage(self) -> ExecutionResources:
+    def min_max_resource_usage_bounds(
+        self,
+    ) -> Tuple[ExecutionResources, ExecutionResources]:
         raise NotImplementedError
```

Comment:
Let's make sure there's a default implementation (so that it doesn't break any custom operators)

Reply:
See my other comment:

https://github.com/ray-project/ray/pull/52502#discussion_r2053342845

Reply:
Responded in the thread for https://github.com/ray-project/ray/pull/52502#discussion_r2053342845